src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -634,57 +628,77 @@ impl A320HydraulicLogic {
         }
     }
 
-    pub fn ptu_ena_output(&self) -> bool {
+    fn ptu_ena_output(&self) -> bool {
         self.ptu_ena_output
     }
-    pub fn rat_on_output(&self) -> bool {
+
+    fn rat_on_output(&self) -> bool {
         self.rat_on_output
     }
-    pub fn blue_electric_pump_output(&self) -> bool {
+
+    fn blue_electric_pump_output(&self) -> bool {
         self.blue_electric_pump_output
     }
-    pub fn yellow_electric_pump_output(&self) -> bool {
+
+    fn yellow_electric_pump_output(&self) -> bool {
         self.yellow_electric_pump_output
     }
-    pub fn green_loop_fire_valve_output(&self) -> bool {
+
+    fn green_loop_fire_valve_output(&self) -> bool {
         self.green_loop_fire_valve_output
     }
-    pub fn yellow_loop_fire_valve_output(&self) -> bool {
+
+    fn yellow_loop_fire_valve_output(&self) -> bool {
         self.yellow_loop_fire_valve_output
     }
-    pub fn engine_driven_pump_2_output(&self) -> bool {
+
+    fn engine_driven_pump_2_output(&self) -> bool {
         self.engine_driven_pump_2_output
     }
-    pub fn engine_driven_pump_1_output(&self) -> bool {
+
+    fn engine_driven_pump_1_output(&self) -> bool {
         self.engine_driven_pump_1_output
     }
-    pub fn green_edp_has_fault(&self) -> bool {
-        self.green_edp_has_fault
+
+    fn green_edp_has_fault(&self) -> bool {
+        // TODO wrong logic: can fake it using pump flow == 0 until we implement check valve sections in each hyd loop
+        // at current state, PTU activation is able to clear a pump fault by rising pressure, which is wrong
```

Comment:
Note that I've repeated this same TODO comment in all `..._has_fault` functions. I'm not sure I understand the comment itself, but for now I guess that's okay.

Reply:
Faults are not like the real thing for now: we should in fact compare two pressures coming from two separated parts of the same hydraulic loop, which is not simulated for now. So we can only "simulate" the fault by that alternative logic of "we drive the pump but still no pressure". Which means if another pump can rise the pressure fault will wrongly be cleared for faulty pump.
That said simpler code indeed thanks!